Makefile

```diff
@@ -61,9 +61,9 @@ publish:
 	rm -rf $(NON_DOCS_FILES)
 	git checkout $(GIT_REF) -- $(NON_DOCS_FILES)
 	$(MAKE) GIT_REF=$(GIT_REF) all
+	git add -A docs
```

Comment:
Moved this here, other the `git clean` was deleting newly created charts... oops!